images/spz/spz/templates/internal/registrations.html

```diff
@@ -22,4 +22,14 @@ <h3 class="ui dividing header">Import</h3>
         <button type="submit" class="fluid ui positive button">Einlesen</button>
     </form>
 </div>
+
+<div class="row">
+    <h3 class="ui dividing header">Matrikelnummer-Überprüfung</h3>
+    <form class="ui form" method="post" action="{{ url_for('registrations_verify') }}" enctype="multipart/form-data">
+        {{ csrf_field() }}
+        {{ render_input(form.number) }}
```

Comment:
Change that to a string input. The reason for this is that we might want to whitelist specific coworker accounts (for KIT that's "a...") so that they don't have to pay. Also, the tag isn't a number in a traditional sense but rather some kind of ID.